### ISSUE TITLE
Feature request: Add logo to the site & colours

### ISSUE THREAD

**Issue by mgifford** [CivicActions/accessibility#246]
**Is your feature request related to a problem? Please describe.**
We should update the site to include our brand

**Comment by dmundra** [comment]
@mgifford I presume we should wait for the new site to be launched? Maybe the homesite folks can give us directions on what colors we should adjust?

**Comment by mikang** [comment]
@mgifford @dmundra I just realized that we will be launching our new brand externally with the new homesite. The new brand hasn't been rolled out publicly. Maybe we _should_ wait for the new site to be launched... @dmundra hmm

Otherwise, I could give some directions on how to make the sites more consistent with each other because I am designing our new homesite. Thoughts?

**Comment by dmundra** [comment]
@mikang yes would love your direction and I can help make the adjustments.

CC: @lukefretwell

**Comment by mgifford** [comment]
@mikang the new logo & colors close enough to the existing ones I don't think it would detract from the main site. Keith would like to start pushing this site though, and so having some nod to our branding would be good.  And ya.. I'm with @dmundra on designs...  

Basics though would be red logo, and some of the background colors.  Might also be a place to explore how we implement the brand on a less important digital product. Anyways, don't want to take from the splash of the home page launch.
